azure-pipelines/templates/create-github-release.yml

```diff
@@ -28,6 +46,7 @@ jobs:
             $(Agent.BuildDirectory)/dist/bedrock-linux
             $(Agent.BuildDirectory)/dist/bedrock-macos
             $(Agent.BuildDirectory)/dist/bedrock-win.exe
+            $(Agent.BuildDirectory)/dist.tar.gz
```

Comment:
@edaena Why do we need this? I think with the current set up this will appear as something to download in the GitHub releases page.